tests/Plugins/utilities.py

```diff
@@ -0,0 +1,68 @@
+# -------------------------------------------------------------------------------
+# |
+# |  Copyright (c) 2024 Scientific Software Engineering Center at Georgia Tech
+# |  Distributed under the MIT License.
+# |
+# -------------------------------------------------------------------------------
+"""Helper utilities for Github tests"""
+
+from git import Repo
+
+import re
+from typing import Match
+from dbrownell_Common.TestHelpers.StreamTestHelpers import ScrubDuration
+
+
+# ----------------------------------------------------------------------
+def ScrubSpaces(content: str) -> str:
+    """
+    Function to remove variable spaces after a match with a Github URL from a string.
+    """
+
+    def replace_func(match: Match) -> str:
+        return match.group("github_url") + "<scrubbed-space>"
+
+    return re.sub(
+        r"(?P<github_url>'https://github.com.+')(\s+)",
+        replace_func,
+        content,
+    )
+
+
+# ----------------------------------------------------------------------
+def ReplaceGithubUrl(content: str) -> str:
+    """
+    Function to replace the GitHub username and (possibly) alternative repository name with the default one.
+    """
+
+    def replace_func(match: Match) -> str:
+        return f"'https://github.com/gt-sse-center/RepoAuditor{match.group('github_path')}"
+
+    return re.sub(
+        r"('https://github.com)/(?P<github_username>[^/]+)/(?P<github_reponame>[^/]+)(?P<github_path>.+')",
```

Comment:
This won't work for GitHub enterprise, as the url won't start with `https://github.com`.